zipline/__main__.py

```diff
@@ -17,6 +18,20 @@
 except NameError:
     __IPYTHON__ = False
 
+extension_args = {}
```

Comment:
How do you feel about making this an object with attribute access so that we can encourage users to namespace their keys like `zipline_live.variable`?